### PR TITLE
chore(admission-controller,cloud-connector,cluster-scanner,node-analyzer,registry-scanner,sysdig-deploy): change charts ownership

### PR DESCRIPTION
**Pull request by mavimo**
## Checklist

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

### CODE REVIEW

File: charts/sysdig-deploy/Chart.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ dependencies:
   - name: node-analyzer
     # repository: https://charts.sysdig.com
     repository: file://../node-analyzer
-    version: ~1.18.3
+    version: ~1.18.4
     alias: nodeAnalyzer
     condition: nodeAnalyzer.enabled
   - name: cluster-scanner
```
Comment (AlbertoBarba) [inline charts/sysdig-deploy/Chart.yaml:42]: You should also bump cluster-scanner

File: charts/admission-controller/Chart.yaml [hunk 2]
```diff
@@ -2,13 +2,13 @@ apiVersion: v2
...
 type: application
-version: 0.14.14
+version: 0.14.15
 appVersion: 3.9.35
 home: https://sysdiglabs.github.io/admission-controller/
 icon: https://avatars.githubusercontent.com/u/5068817?s=200&v=4
 maintainers:
-  - name: giuse-sysdig
-    email: giuseppe.esposito@sysdig.com
+  - name: guidobonomi
```
Comment (guidobonomi) [inline charts/admission-controller/Chart.yaml]: this is not correct: Admission controller is now under platform team and is not in VM. Maybe @acaprari can help identifying the proper ownership for this

File: charts/cloud-connector/Chart.yaml [hunk 3]
```diff
@@ -3,10 +3,10 @@ name: cloud-connector
...
 type: application
-version: 0.8.6
+version: 0.8.7
 appVersion: 0.16.48
 home: https://sysdiglabs.github.io/cloud-connector
 
 maintainers:
-  - name: giuse-sysdig
-    email: giuseppe.esposito@sysdig.com
+  - name: guidobonomi
```
Comment (guidobonomi) [inline charts/cloud-connector/Chart.yaml]: this is not correct as well. This should go under @acaprari  please

Comment (acaprari) [reply charts/cloud-connector/Chart.yaml]: 👍🏼 @mavimo you can reassign to me

File: charts/registry-scanner/Chart.yaml [hunk 4]
```diff
@@ -2,12 +2,11 @@ apiVersion: v2
...
-appVersion: 0.2.58
+home: https://www.sysdig.com/
+icon: https://avatars.githubusercontent.com/u/5068817?s=200&v=4
+version: 1.1.22
+appVersion: 0.2.59
 maintainers:
-  - name: giuse-sysdig
-    email: giuseppe.esposito@sysdig.com
+  - name: sysdiglabs
   - name: hayk
```
Comment (guidobonomi) [inline charts/registry-scanner/Chart.yaml]: I'd suggest to remove hayk as well

**Review by acaprari — APPROVED** [review]

**Pull request merged**